src/controllers/authController.js

```diff
@@ -16,4 +17,41 @@ export default class AuthController {
       res.status(400).json({ status: 'error', error });
     }
   }
+
+  /**
+ *
+ * @param {*} res - response object
+ * @return {object} - user data and status code
+ */
+  static async signupUser({ body }, res) {
+    try {
+      const data = await AuthService.signup(body);
+
+      await send(data);
+
+      res.status(201).json({ status: 'success', data });
+    } catch (error) {
+      res.status(409).json({ status: 'error', error });
+    }
+  }
+
+  /**
+  @static
+  * @description Returns message based on the status
+  * @param {*} req - request object
+  * @param {*} res - response object
+  * @return {object} - message
+  * @memberof AuthController
+  */
+  static async verify(req, res) {
+    try {
+      const is_verified = await AuthService.verifyUser(req, res);
+      if (is_verified) {
+        return res.status(200).json({ message: 'Email Successfully Verified' });
+      }
+    } catch (error) {
+      const { status, message } = error;
+      return Helper.errorstatus(res, status, `${message}`);
```

Comment:
please use this format
`
res.status(400).json({ status: 'error', message });
`

instead of this 
`
return Helper.errorstatus(res, status, `${message}`);
`

There is no need for this abstraction